content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -83,10 +80,8 @@ $excerpt-line-height: 20;
   }
 
   .meta {
-    display: flex;
-    flex-direction: column;
```

Comment:
we need flex-direction to align context 'sponsored by..'. Only CSS unused and safe to remove is flex-grow:1